docs/react/start-streaming.mdx

```diff
@@ -32,38 +32,37 @@ The code above will make the browser ask the user for camera and microphone perm
 
 To manage users' camera and microphone devices, use the respective [`useCamera`](/api/web/functions/useCamera)
 and [`useMicrophone`](/api/web/functions/useMicrophone) hooks.  
-Both of them has the same API. To keep things simple, let's just use the camera hook.
+Both of them has similar API. To keep things simple, we will just use the camera hook.
 
 ```tsx
 import { useEffect, useRef } from "react";
 import { useCamera } from "@fishjam-cloud/react-client";
 
-export function ExampleComponent() {
+export function ExampleCameraPreview() {
   const videoRef = useRef<HTMLVideoElement>(null);
 
   // highlight-next-line
-  const { isStreaming, activeDevice, initialize, stream, devices } =
+  const { activeCamera, selectCamera, cameraStream, cameraDevices } =
     // highlight-next-line
     useCamera();
 
   useEffect(() => {
     if (!videoRef.current) return;
-    videoRef.current.srcObject = stream ?? null;
+    videoRef.current.srcObject = cameraStream ?? null;
   }, [stream]);
```

Comment:
```suggestion
  }, [cameraStream]);
```

(might be worth to paste these examples into editor with eslint/prettier enabled)

Reply:
processed it with gpt